mscore/preferences.h

```diff
@@ -27,6 +27,8 @@
 
 namespace Ms {
 
+extern QString mscoreGlobalShare;
```

Comment:
ANSWER: fluid synth needs that string.  Here is compiler error when I remove that line:

<pre>/home/e/MuseScore/fluid/fluid.cpp: In static member function 'static QFileInfoList FluidS::Fluid::sfFiles()':
/home/e/MuseScore/fluid/fluid.cpp:895:48: error: 'mscoreGlobalShare' was not declared in this scope
       pl.prepend(QFileInfo(QString("%1%2").arg(mscoreGlobalShare).arg("sound")).absoluteFilePath());
                                                ^
fluid/CMakeFiles/fluid.dir/build.make:111: recipe for target 'fluid/CMakeFiles/fluid.dir/fluid.cpp.o' failed
make[2]: *** [fluid/CMakeFiles/fluid.dir/fluid.cpp.o] Error 1
make[1]: *** [fluid/CMakeFiles/fluid.dir/all] Error 2
make: *** [all] Error 2</pre>